src/index.html

```diff
@@ -2,11 +2,348 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0" name="viewport">
+    <meta content="ie=edge" http-equiv="X-UA-Compatible">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com" rel="preconnect">
+    <link href="https://fonts.gstatic.com" rel="preconnect">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link href="./styles/main.scss" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+
+    <header class="header">
+      <a
+        class="header__logo"
+        href="/"
+      >
+        <img
+          alt="MOYO logo"
+          src="images/logo.png"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active"
+               href="#apple">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link"
+               href="#samsung">Samsung</a>
```

Comment:
Need to fix it everywhere
```suggestion
            <a class="nav__link" href="#samsung">Samsung</a>
```